samples/notebooks/powershell/Docs/Profile Support.ipynb

```diff
@@ -0,0 +1,131 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# `$PROFILE` Support in a PowerShell Notebook <img src=\"https://raw.githubusercontent.com/PowerShell/PowerShell/master/assets/Powershell_black_64.png\" align=\"right\"/>\n",
+    "\n",
+    "For a while now, PowerShell has had this concept of \"Profiles\" which are scripts that run when a new instance of PowerShell is started. Inside of a profile, you can put any PowerShell script you'd like.\n",
+    "\n",
+    "To see the location of the profile, simply run:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "$PROFILE"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can create that file and fill it with anything you want to run before your Notebook runs the first cell of the session.\n",
+    "\n",
+    "This profile path is specific to .NET Interactive and specific to the current user who's running it.\n",
+    "\n",
+    "### Give it a try\n",
+    "\n",
+    "Let's add a definition of a global variable to our profile:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "'$global:SET_IN_PROFILE = $true' >> $PROFILE"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "_Now open a new Notebook and try running `$global:SET_IN_PROFILE`. It should be set to `True`!_\n",
```

Comment:
It's not rendered as italic in the gist, not sure if it's just the gist view or something should be fixed.

Reply:
looks like it likes * instead of _ - fixed!